lib/module.ml

```diff
@@ -395,11 +395,11 @@ and reflect_named ppf (n,m) = Pp.fp ppf "(%S,%a)" n reflect m
 and reflect_namespaced ppf nd =
   if nd.namespace = [] then
     Pp.fp ppf "Namespaced.make %a"
-      Pp.estring nd.name
+      Modname.pp nd.name
```

Comment:
`reflect` functions are supposed to generate valid OCaml code, this is not the case here.